src/examples/examples.csproj

```diff
@@ -1,7 +1,7 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFrameworks>net6.0;net7.0</TargetFrameworks>
+    <TargetFrameworks>net8.0;net9.0</TargetFrameworks>
```

Comment:
We should likely match https://github.com/criipto/criipto-signatures-sdk/blob/master/packages/dotnet/Criipto.Signatures/Criipto.Signatures.csproj#L3